.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report an issue with existing supported devices.
+labels: ["bug"]
+body:
```

Comment:
The order in which each information is requested was intentional: a good description of the bug is the most important part, followed by the output from all relevant commands, and then the additional information.

Reply:
I moved the entries around to reflect your comment and the current template.